docs/user-manual/en/amqp-server-DR.md

```diff
@@ -0,0 +1,9 @@
+# AMQP and Multiple Replicas
+
+After 2.16.0 ActiveMQ Artemis added the ability to capture broker events:
+
+You can 
```

Comment:
this seems half finished?

Reply:
the doc still under work.. I will remove this chapter.